### ISSUE TITLE
Installation of git package fails to update mandoc.db

### ISSUE THREAD

**Issue by zqqw** [voidlinux/void-packages#3943]
After installing Git:
$ sudo xbps-install git
Name Action    Version           New version            Download size
git  install   -                 2.7.4_1                3906KB
Size to download:             3907KB
Size required on disk:          29MB

$ man git-var
man: outdated mandoc.db lacks git-var(1) entry,
     consider running  # makewhatis /usr/share/man
$ sudo makewhatis /usr/share/man

**Comment by Vaelatern** [comment]
```
<Vaelatern>  Should we add a hook to makewhatis if a manpage is installed?
<Vaelatern>  Wouldn't help current packages, but would slowly fix all of them
<duncaen>    or just run makewhatis through the daily cron
<Vaelatern>  We have no control over user's daily cron
<duncaen>    but its already there and we dont have crontrol over which man someone uses
<duncaen>    ok i guess there is only mdocml
<duncaen>    there is man-db too
<Vaelatern>  Huh mdocml-1.13.3_6 /etc/cron.daily/makewhatis
<Vaelatern>  We can call makewhatis on only certain files...
<duncaen>    Vaelatern: would not be bad i guess, you could check if /usr/bin/man is a link to mandoc or mandb-man to call the right update commands
<duncaen>    but xbps-alternatives would stiff fiddle with the man pages and you would need to update manually
<duncaen>    s/stiff/still
<Vaelatern>  But by default it would work
<duncaen>    y
```

**Comment by zqqw** [comment]
I found another instance:
$ man zathura
man: outdated mandoc.db lacks zathura(1) entry,
     consider running  # makewhatis /usr/share/man
Keeping in mind I hadn't even heard of makewhatis until I got these messages...
using aptitude in other distros there would be a message onscreen during package installation "updating man pages" or similar. An unnecessary cron job wastes CPU power, if it's updated when a package with a man page is installed it'd be more efficient. Current users getting the message to run makewhatis should figure it out, and presumably if it was fixed there, when they next install (or perhaps update?) a package with a manpage, mandocdb would be updated.
